### PR TITLE
add discord verification, how to delete and how to edit a domain

### PR DESCRIPTION
**Pull request by Stef-00012**

### CODE REVIEW

File: src/docs/delete_domain/index.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# How to delete your is-a.dev domain
+
+## Open your fork of the [is-a-dev/register](https://github.com/is-a-dev/register) repository
+
+1. Open your fork, or if you deleted it, [fork](https://github.com/is-a-dev/register/fork) again the repository
```
Comment (wdhdev) [inline src/docs/delete_domain/index.md:5]:
```suggestion
1. Open your fork, or if you deleted it, [fork](https://github.com/is-a-dev/register/fork) the repository once again.
```

File: src/docs/delete_domain/index.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+2. Find you domain file in the `domains` folder (`/domains/<subdomain>.json`)
```
Comment (wdhdev) [inline src/docs/delete_domain/index.md:6]:
```suggestion
2. Find your domain file in the `domains/` folder (e.g. `/domains/myname.json`)
```

File: src/docs/delete_domain/index.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+3. Press the 3 dots and press the "Delete file" button
```
Comment (wdhdev) [inline src/docs/delete_domain/index.md:7]:
```suggestion
3. Press the 3 dots and press the `Delete file` button.
```

**Comment by Stef-00012** [comment]
also removed twitter field in cf pages and domain structure

File: src/docs/delete_domain/index.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+<img src="../img/delete_domain_step_1.png" height="250">
+
+4. Press the button "Commit changes"
```
Comment (wdhdev) [inline src/docs/delete_domain/index.md:11]:
```suggestion
4. Press `Commit changes`.
```

File: src/docs/delete_domain/index.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+
+<img src="../img/delete_domain_step_2.png" height="250">
+
+5. In the popup that appears, press the button "Commit changes"
```
Comment (wdhdev) [inline src/docs/delete_domain/index.md:15]:
```suggestion
5. In the popup that appears, press `Commit changes` once again.
```

File: src/docs/delete_domain/index.md [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+
+<img src="../img/delete_domain_step_3.png" height="250">
+
+6. Press the "Contribute" button and then press on the "Open pull request" button
```
Comment (wdhdev) [inline src/docs/delete_domain/index.md:19]:
```suggestion
6. Click the `Contribute` button, then press `Open pull request`.
```

File: src/docs/delete_domain/index.md [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+
+<img src="../img/delete_domain_step_4.png" height="250">
+
+7. Press the "Create pull request" button
```
Comment (wdhdev) [inline src/docs/delete_domain/index.md:23]:
```suggestion
7. Press `Create pull request` again.
```

File: src/docs/delete_domain/index.md [hunk 8]
```diff
@@ -0,0 +1,27 @@
...
+
+<img src="../img/delete_domain_step_5.png" height="250">
+
+And you're done! After you pull request has been merged, the domain will be deleted
```
Comment (wdhdev) [inline src/docs/delete_domain/index.md:27]:
```suggestion
And you're done! After you pull request has been merged, the domain will be deleted.
```

File: src/docs/discord_verification/index.md [hunk 9]
```diff
@@ -0,0 +1,47 @@
+# Setting up Discord domain connection with your is-a.dev domain
+
+## Get your verification string
+
+1. Open your settings
```
Comment (wdhdev) [inline src/docs/discord_verification/index.md:5]:
```suggestion
1. Open your Discord app and press `Settings`.
```

File: src/docs/discord_verification/index.md [hunk 10]
```diff
@@ -0,0 +1,47 @@
...
+
+<img src="../img/discord_step_1.png" height="259">
+
+2. Open the "Connections" section
```
Comment (wdhdev) [inline src/docs/discord_verification/index.md:9]:
```suggestion
2. Open the `Connections` section.
```

File: src/docs/discord_verification/index.md [hunk 11]
```diff
@@ -0,0 +1,47 @@
...
+
+<img src="../img/discord_step_2.png" height="259">
+
+3. Press the "View more" button
```
Comment (wdhdev) [inline src/docs/discord_verification/index.md:13]:
```suggestion
3. Press the `View more` button.
```

File: src/docs/discord_verification/index.md [hunk 12]
```diff
@@ -0,0 +1,47 @@
...
+
+<img src="../img/discord_step_3.png" height="259">
+
+4. Click on the domain button (a globe icon)
```
Comment (wdhdev) [inline src/docs/discord_verification/index.md:17]:
```suggestion
4. Click on the domain button (the globe icon).
```

File: src/docs/discord_verification/index.md [hunk 13]
```diff
@@ -0,0 +1,47 @@
...
+
+<img src="../img/discord_step_4.png" height="259">
+
+5. In the field that appears type your is-a.dev domain name
```
Comment (wdhdev) [inline src/docs/discord_verification/index.md:21]:
```suggestion
5. In the field that appears type your is-a.dev domain name (e.g. `myname.is-a.dev`).
```

File: src/docs/discord_verification/index.md [hunk 14]
```diff
@@ -0,0 +1,47 @@
...
+
+<img src="../img/discord_step_5.png" height="259">
+
+6. Copy the verification string
```
Comment (wdhdev) [inline src/docs/discord_verification/index.md:25]:
```suggestion
6. Copy the verification string.
```

File: src/docs/discord_verification/index.md [hunk 15]
```diff
@@ -0,0 +1,47 @@
...
+
+<img src="../img/discord_step_6.png" height="259">
+
+## Create the domain file
+
+Create a JSON file inside the `domains` directory (`domains/_discord.<subdomain>.json`) with the following content:
```
Comment (wdhdev) [inline src/docs/discord_verification/index.md:31]:
```suggestion
Create a JSON file inside the `domains/` directory called `domains/_discord.yourname.json` with the following content:
```

File: src/docs/discord_verification/index.md [hunk 16]
```diff
@@ -0,0 +1,47 @@
...
+
+```json 
+{
+    "owner": {
+        "username": "<github-username>",
```
Comment (wdhdev) [inline src/docs/discord_verification/index.md:36]:
```suggestion
        "username": "github-username",
```

File: src/docs/discord_verification/index.md [hunk 17]
```diff
@@ -0,0 +1,47 @@
...
+        "email": "email@address"
+    },
+    "record": {
+        "TXT": "<discord-verification-string>"
```
Comment (wdhdev) [inline src/docs/discord_verification/index.md:40]:
```suggestion
        "TXT": "discord-verification-string"
```

File: src/docs/cloudflare_pages/index.md [hunk 18]
```diff
@@ -14,8 +14,7 @@ Create a JSON file inside `domains` directory (`domains/<subdomain>.json`) with
 {
     "owner": {
         "username": "<github-username>",
```
Comment (wdhdev) [inline src/docs/cloudflare_pages/index.md:16]:
```suggestion
        "username": "github-username",

```

File: src/docs/discord_verification/index.md [hunk 19]
```diff
@@ -0,0 +1,47 @@
...
+    }
+} 
+```
+
+# Configuration
+After your pull request has been merged, repeat the steps to get the verification string and press the "Verify" button.
```
Comment (wdhdev) [inline src/docs/discord_verification/index.md:46]:
```suggestion
After your pull request has been merged, repeat the steps to get the verification string and press the `Verify` button.
```

File: src/docs/discord_verification/index.md [hunk 20]
```diff
@@ -0,0 +1,47 @@
...
+If it shows any error such as "Unable to verify your domain", try waiting a few minutes, the DNS change might not have reached discord DNS server yet
```
Comment (wdhdev) [inline src/docs/discord_verification/index.md:47]:
```suggestion
If it shows any error such as `Unable to verify your domain`, try waiting a few minutes (sometimes up to 24 hours) as the DNS change might not have reflected on the DNS server.
```

File: src/docs/edit_domain/index.md [hunk 21]
```diff
@@ -0,0 +1,35 @@
...
+
+7. Press the "Contribute" button and then press on the "Open pull request" button
+
+<img src="../img/edit_domain_step_5.png" height="250">
+
+8. Press the "Create pull request" button
+
+<img src="../img/edit_domain_step_6.png" height="250">
+
+9. Add a title to your pull request and then press the "Create pull request button"
```
Comment (wdhdev) [inline src/docs/edit_domain/index.md:31]:
```suggestion
8. Complete the pull request template in the description field, add a title to the pull request, then press `Create pull request`.
```

**Review by wdhdev — CHANGES_REQUESTED** [review]

**Review by wdhdev — APPROVED** [review]

**Pull request merged**